src/Microsoft.AspNetCore.Mvc.WebApiCompatShim/Conventions/IUseWebApiActionConventions.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace Microsoft.AspNetCore.Mvc.WebApiCompatShim
 {
+    /// <summary>
+    /// Indicates actions without attribute routes in a controller use WebAPI routing conventions.
```

Comment:
> "web API" generic concept, or the specific product "ASP.NET Web API." Which is it?

It's the general concept because the conventions make significantly more sense there than in a views world. It's the specific product because we're mimicking ASP.NET Web API 3.x in this shim and with these particular conventions. Let's go w/ "web API".

Any objection?